.drone.yml

```diff
@@ -32,6 +37,48 @@ steps:
     commands:
       - ./gradlew --no-daemon dependencyUpdates
 
+  - name: dev - assume role
```

Comment:
Should this step also occur on master only?

Reply:
Indeed. Thanks!